pkg/queue/queue.go

```diff
@@ -15,7 +15,8 @@ var (
 )
 
 const (
-	defaultQueueName = "Queue"
+	loggingRateLimiter = "pkg-queue"
```

Comment:
It looks like this is sort of a key, not the limiter itself, but I see it is named like this in the entire code, so okay to keep it.

Reply:
Yeah. I basically copied the way it's named in other places for consistency.